Documentation/contributing/release/backports.rst

```diff
@@ -140,6 +140,17 @@ Creating the backports branch
       ...
       $ contrib/backporting/cherry-pick <newest-commit-sha>
 
+   Conflicts may be resolved by applying changes or backporting other
+   PRs to completely avoid conflicts. Backporting entire PRs is preferred if the changes in the dependent PRs are small. `This
```

Comment:
Nit: This is missing a newline, although I'm not sure why we seem to enforce a max. line width for the documentation.